pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSources.java

```diff
@@ -207,10 +201,18 @@ void runCmd() throws Exception {
 
         @Parameter(names = "--processingGuarantees", description = "The processing guarantees (aka delivery semantics) applied to the Source")
         protected FunctionConfig.ProcessingGuarantees processingGuarantees;
-        @Parameter(names = "--destinationTopicName", description = "The Pulsar topic to which data is sent")
+
+        @Parameter(names = { "-o", "--destinationTopicName" }, description = "The Pulsar topic to which data is sent")
         protected String destinationTopicName;
-        @Parameter(names = "--deserializationClassName", description = "The SerDe classname for the source")
+
+        @Parameter(names = "--deserializationClassName", description = "The SerDe classname for the source", hidden = true)
         protected String deserializationClassName;
```

Comment:
shouldn;t this be DEPRECATED_deserialzationClassName

Reply:
That will come in @jerrypeng PR